test/link.test.js

```diff
@@ -3,6 +3,20 @@ import { render, cleanup, fireEvent } from "@testing-library/react";
 
 import { Router, Link } from "../index.js";
 
+const ParentComponent = (props) => {
```

Comment:
Can we rename this to something like `LinkWithForwardedRef` or similar just to avoid confusion with the rest of the tests?

Reply:
Sure